lib/servers.json

```diff
@@ -96,5 +102,11 @@
         "t": "50001",
         "version": "1.4"
     }
+    "jh3jgcrwweh6yvmprtjnp72u2hqn34nlftlg3msrr4vmlapft4yvt2id.onion": {
```

Comment:
You are missing a comma here and in other places.  Json parsers will reject this file if I merge this as-is.